scripts/globals/helm.lua

```diff
@@ -1389,10 +1391,20 @@ local helmInfo =
 }
 
 -----------------------------------
--- colored rocks. do not change this order!
+-- colored rocks array
 -----------------------------------
 
-local rocks = { 769, 771, 770, 772, 773, 774, 776, 775 }
+local rocks =
+{
+    [xi.magic.element.FIRE] = xi.items.RED_ROCK,
+    [xi.magic.element.ICE] = xi.items.TRANSLUCENT_ROCK,
+    [xi.magic.element.WIND] = xi.items.GREEN_ROCK,
+    [xi.magic.element.EARTH] = xi.items.YELLOW_ROCK,
+    [xi.magic.element.THUNDER] = xi.items.PURPLE_ROCK,
+    [xi.magic.element.WATER] = xi.items.BLUE_ROCK,
+    [xi.magic.element.LIGHT] = xi.items.WHITE_ROCK,
+    [xi.magic.element.DARK] = xi.items.BLACK_ROCK,
+}
```

Comment:
Dug a little deeper, and I think there's an error in the logic as well.
```
    -- if we picked a colored rock, change it to the day's element
    if item == 769 then
        item = rocks[VanadielDayElement() + 1]
    end
```

This shouldn't be + 1, since we should never get a none-type daily element

Reply:
probably leftover from the old methods of aligning element and day since their orders didn't match back then and they didn't start at the same value

Reply:
That was actually me screwing up on git. I had originally fixed that but when I squashed I lost the change somehow.